src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1755,6 +1789,26 @@ private void emitNoResolvedType(NoType type, String defaultEmit) {
       }
     }
 
+    private String getDisplayNameFromType(JSType type) {
+      String displayName = type.getDisplayName();
+      // In partial mode, closure doesn't know the correct name of imported symbols, if the name
```

Comment:
Sentence splice makes this a bit weird to read.  Maybe change hte second comma to a period?

Reply:
Done.